go/cmd/vtctldclient/command/vreplication/vdiff/vdiff.go

```diff
@@ -57,13 +57,13 @@ var (
 		TargetCells                 []string
 		TabletTypes                 []topodatapb.TabletType
 		Tables                      []string
-		Limit                       uint32 // We only accept positive values but pass on an int64
+		Limit                       int64
```

Comment:
If we want to accept only positive values, should this not be `uint64`?? Either that, or we'll need to add validation.

Reply:
That was the idea initially, but vtctlclient allows it and 0 is equivalent to any negative value. We can't use uint64 as we'll then allow larger values than the backend supports. I was on the fence so left it. I can add a quick check for positive values though if you like that more.

Reply:
What does it mean that "0 is equivalent to any negative value"? What behavior do you get with a value of 0 for any of these flags?

Reply:
As an example: https://github.com/vitessio/vitess/blob/main/go/vt/vttablet/tabletmanager/vdiff/table_differ.go#L534-L538

But no matter as I added checks to prevent negative values from the client.